<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[*] TDPVista - TextEditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/images/app_icon.png</normaloff>:/images/images/app_icon.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Selection</string>
    </property>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuTerminal">
    <property name="title">
     <string>Terminal</string>
    </property>
    <addaction name="actionNew_Terminal"/>
    <addaction name="actionSplit_Terminal"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWelcome"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSelection"/>
   <addaction name="menuView"/>
   <addaction name="menuTerminal"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/new_file.png</normaloff>:/images/images/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/open_file.png</normaloff>:/images/images/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/save_file.png</normaloff>:/images/images/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/save_as_file.png</normaloff>:/images/images/save_as_file.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/images/exit_app.png</normaloff>:/images/images/exit_app.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset theme="edit-find-replace">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Find/Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionNew_Terminal">
   <property name="text">
    <string>New Terminal</string>
   </property>
  </action>
  <action name="actionSplit_Terminal">
   <property name="text">
    <string>Split Terminal</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="text">
    <string>Welcome</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectAll</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
